Server/ConcordiaCurriculumManager/appsettings.Development.json

```diff
@@ -13,7 +13,7 @@
     "Audience": "ccm-backend"
   },
   "DatabaseSettings": {
-    "ConnectionString": "Host=localhost;Port=5432;Database=ConcordiaCurriculumManager;Username=postgres;Password=password"
+    "ConnectionString": "Host=localhost;Port=5432;Database=ConcordiaCurriculumManager;Username=postgres;Password=postgres"
```

Comment:
You didn't mean to commit this, did you?

Reply:
nope